master/internal/db/postgres_command.go

```diff
@@ -46,7 +46,7 @@ func IdentifyTask(ctx context.Context, taskID model.TaskID) (TaskMetadata, error
 	metadata := TaskMetadata{}
 	if err := Bun().NewSelect().Model(&metadata).
 		ColumnExpr("generic_command_spec->'Metadata'->'workspace_id' AS workspace_id").
-		ColumnExpr("generic_command_spec->'TaskType' as task_type").
+		ColumnExpr("generic_command_spec->>'TaskType' as task_type").
```

Comment:
question: why do you want to return only this column as text?

Reply:
also, would it be worth moving `postgres_command` from the db module to the command module -- I know we do this for user/usergroup modules already & I do like that organization. Open a backlog ticket perhaps?

Reply:
I reverted the return only this column as text and instead created a ticket

https://hpe-aiatscale.atlassian.net/browse/DET-10004

to describe the bug since it is customer facing and will need a release note and can keep this PR test only changes 

but tldr ```generic_command_spec->'TaskType' as task_type``` gives us extra quotes ```"TASK_TYPE_COMMAND"``` instead of ```TASK_TYPE_COMMAND```